config/prod.exs

```diff
@@ -26,6 +26,12 @@ config :castle, :bigquery, BigQuery
 config :castle, :rollup_initial_delay, 60
 config :castle, :rollup_delay, 300
 
+# Scheduled jobs
+config :castle, Castle.Scheduler,
+  jobs: [
+    {"* * * * *", {Mix.Tasks.Feeder.Sync, :run, [["--lock"]]}},
```

Comment:
Here's the magic!  In prod, this task will run every minute, with the `--lock` flag to ensure multiple servers aren't hitting Feeder with the same queries.